examples/chart/teleport-kube-agent/templates/deployment.yaml

```diff
@@ -119,18 +119,17 @@ spec:
         {{- if .Values.imagePullPolicy }}
         imagePullPolicy: {{ toYaml .Values.imagePullPolicy }}
         {{- end }}
-        {{- if or .Values.extraEnv .Values.tls.existingCASecretName }}
         env:
+        # This variable is set for telemetry purposes.
+        # Telemetry is opt-in and controlled at the auth level.
```

Comment:
nit: it's not technically opt-in, it's always enabled, but only for cloud (and eventually enterprise) users

Reply:
```suggestion
        # Telemetry is opt-in for oss users and controlled at the auth level.
```